blocks/header/header.js

```diff
@@ -35,16 +38,19 @@ function getNavbarToggler() {
   }
```

Comment:
Just wondering what is the need of adding the above styling in js.
Not related to your change but still want to know

Reply:
When the window is downsized, the nav items are displayed inside the mobile-icon toggler. It seems it was added as an placeholder for navbar toggler.